### ISSUE TITLE
CQ's not working at all in 0.9 releases

### ISSUE THREAD

**Issue by marcosnils** [influxdb/influxdb#2402]
I've tested this from rc_23 to rc_27 and I can't get it to work.

Use case: 

select * from cpu_load_short:
```
name: cpu_load_short
--------------------
time			value
2014-11-10T16:00:00Z	0.64
2014-11-10T17:00:00Z	0.64
2014-11-10T18:00:00Z	0.64
2014-11-10T19:00:00Z	0.64
2014-11-10T20:00:00Z	0.64
2014-11-10T21:00:00Z	0.64
2014-11-10T22:00:00Z	0.64
2014-11-10T23:00:00Z	0.64
2014-12-10T23:00:00Z	0.8
```

create continuous query "test_cq" on segments begin select count(value) into "downsample" from cpu_load_short where time > now() - 170d and time < now() - 150d group by time(1m) fill(0) end

CQ get's created successfully and then nothing happens, I've set up the loglevel in "debug" and still not seeing anything. 

Here's my CQ's config:

```
[continuous_queries]
  recompute-previous-n = 2
  recompute-no-older-than = "10m0s"
  compute-runs-per-interval = 10
  compute-no-more-than = "2m0s"
  disabled = false
```

**Comment by wildchild** [comment]
Try the following setup: run influxdb without authentication and push points to database without artificial "timestamp" and "precision". I got it working finally today. 1st step or 2nd should help. I tested these two weird conditions and not sure for now what was right condition to make CQ running.

**Comment by marcosnils** [comment]
@wildchild my db doesn't have auth. I'll try posting the events without timestamp (so influx resolves it) and let u know how it goes

**Comment by marcosnils** [comment]
@wildchild still no luck. Which version are you using?. The CQ get's created but the underlying downsampled measurements don't. It's like if the CQ never ran

**Comment by wildchild** [comment]
rc_27. Try to create query similar to query in my last issue. You query is different than my, I have no time conditions and fill(0). Also I specified hostname = "localhost", not sure if it necessary. All this look like alchemy now. CQ must be covered with tests.

**Comment by marcosnils** [comment]
@wildchild bleh!!.. too much magic to make this work, maybe someone from influx can give us a real status about CQ's in 0.9
@otoolep @corylanou @beckettsean ??

**Comment by corylanou** [comment]
We are taking another look at this internally.  There seem to be several related issues with CQ's not working.  We are building out the test suite to give this more coverage which should stabilize this feature. Thanks for your support and patience.

**Comment by marcosnils** [comment]
Thx for the update!.

**Comment by beckettsean** [comment]
Believed fixed in RC28, can you repro?

**Comment by marcosnils** [comment]
@beckettsean sure, I'll give it a try and post my results

**Comment by marcosnils** [comment]
@beckettsean still doesn't work.

**Comment by beckettsean** [comment]
@marcosnils thanks for trying again. We'll dig deeper.

**Comment by beckettsean** [comment]
Almost certainly fixed by #2552, which is in RC30 being released today.
